google-cloud-secret_manager/samples/snippets.rb

```diff
@@ -106,6 +106,39 @@ def create_secret project_id:, secret_id:
   secret
 end
 
+def create_ummr_secret project_id:, secret_id:, locations:
```

Comment:
Did you intend to include a region tag?

Reply:
Yes, this is basically sample to create a secret with one or more locations (regions) specified by user to use for secret replication

Reply:
@kapishps I meant, this sample is missing the appropriate region tag comments to identify the ID and range of the snippet for inclusion in the cloud website. Please see internal link: go/code-snippets-101#region-tag-comments

Reply:
Hi @dazuma we don't intend to include this in cloud website for now. Just adding the snippets in github.